src/main/java/yevhenii/lostfilmdemo/convertors/TVRecordConvertor.java

```diff
@@ -45,12 +51,28 @@ public TvSeriesRecord createRecord(TVSeries tvSeries) {
         record.setLastupdate(tvSeries.getLastUpdate());
         if (tvSeries.getImdbEpisode() == null) return record;
         record.setTitle(tvSeries.getImdbEpisode().getTitle());
-        if (!Objects.equals(tvSeries.getImdbEpisode().getYear(), "N/A"))
-            record.setYear(Integer.parseInt(tvSeries.getImdbEpisode().getYear()));
+        record.setYear(getYear(tvSeries));
         record.setPlot(tvSeries.getImdbEpisode().getPlot());
-        if (!Objects.equals(tvSeries.getImdbEpisode().getImdbRating(), "N/A"))
-            record.setImdbrating(Double.parseDouble(tvSeries.getImdbEpisode().getImdbRating()));
+        record.setImdbrating(getImdbRating(tvSeries));
 
         return record;
     }
+
+    private int getYear(TVSeries tvSeries) {
+        try {
+            return Integer.parseInt(tvSeries.getImdbEpisode().getYear());
+        } catch (NumberFormatException e) {
+            log.debug(e.getMessage());
```

Comment:
this is error so log it as error and pass exception to get full stacktrace in your logs